pkg/download/downloader.go

```diff
@@ -162,20 +163,44 @@ func GetInsecure(uri, dir string, fetcher Fetcher) error {
 	return extractArchive(name, dir, body, size)
 }
 
-func extractArchive(filename, dst string, r io.ReaderAt, size int64) error {
+func extractContentType(at io.ReaderAt) (string, error) {
```

Comment:
call this `detectContentType` or `detectMIMEType`. it's not extracting anything.

also rename `at` to `r`. (ditto in the other method)